### DISCUSSION TITLE
Commission based referring of investors

### DISCUSSION THREAD

**Discussion by nigelhooijmans** [deso-protocol/dips#18] [Medium CIPs]
### Referral commission for bringing in new investors

This could be handy if someone has a large following, but hasn't invested (yet) in the account that he or she promotes, but still wants to make a profit of the large following the creator has. (There could be more than one reason for that e.g. giving others the opportunity to invest at low cost, or making money without investing first.)

The referral commission can be set-up in the profile settings of the BitClout account.

1. Create a checkbox for Referral commission > On/Off
2. % box > determine commission x% (e.g. 6%)

When visiting the BitClout profile that has the Referral commission check ON, a custom link to promote the account is shown.

Use the referral link in a post, and when someone invests, you'll see the commission come in. 
The commission should be paid in the Creator Coin of the promoted account.
If you have chosen e.g. 6% as the referral commission, your received investments will be 6% lower and the 6% goes to the referrer.

Can we BUIDL this? Would this be a good idea?

**Comment by tijno** [comment]
I am not in favour of financial commission:

* I am pretty sure there are some serious legal & compliance issues with commissions for investment. Im not familiar with the investment space but I am sure others are and can clarify.
* Recommending for reward, bonus and commission often drives the wrong behaviours

Of course the benefit of having it would be that its transparant/visible. I am sure some "compensation" is given now for "mentioning" an account - and its all hidden and not disclosed.

And some kind of referral metrics would be hugely beneficial for projects on bitclout.

**Reply by nigelhooijmans** [reply]
Thanks for replying Tijn! 

- legal & compliance issues: yes, I think this might be an issue we're facing here. I'm neither an expert on this area.
- recommending for reward influencing behaviour: yes, it does influence behaviour. But it can also be an extra push for creators with a large following to highlight some smaller creators. Large creators won't probably put this function ON, I think it's the smaller creator that's profiting from this referral system. 

Though I think the behaviour of some on BitClout is already influenced by diamonds and investments. 
BitClout is not your average social media platform, therefore I thought let's give this idea a try here.

In addition, referral metrics would be beneficial for projects, and companies and creators - imo.

**Reply by HPaulson** [reply]
As I read this, All I imagined was multiple government agencies serving lawsuits lmao. It seems like an okay idea in nature but seems very very dangerous in practice. I'm curious if someone more knowledgeable has specific info on this, but seems like something that'd have huge legal Implications (As @Tijn also mentions above)

**Reply by bloated-devish** [reply]
What is the legal implication? I think referral bonus is something pretty standard for any system. Who funds it is the question - anyone in the system can do it. Infact, if you raise a new funds to do this, I am sure you'll get enough folks willing to donate - as this grows the overall ecosystem.

**Reply by tijno** [reply]
Not in financial services and stocks.

Especially when claiming "gains" or "profit" from investments.

Just look at the disclaimers at the bottom of any article, podcast or video recommending a normal financial product, or investment.

Im no lawyer, and laws in each country are different - but i do know this is a minefield.

**Reply by tijn** [reply]
> As I read this, All I imagined was multiple government agencies serving lawsuits lmao. It seems like an okay idea in nature but seems very very dangerous in practice. I'm curious if someone more knowledgeable has specific info on this, but seems like something that'd have huge legal Implications (As @tijn also mentions above)

Could you guys take care to name the right Tijn, please? The one with an extra o: @tijno, is the one you need.(@HPaulson in particular since you mentioned me several times in the last few days.) I understand that typos and small mistakes exists; it's fine, no bad feelings, but for me all these messages are basically spam in my inbox and I have no means to stop it other than hitting the unsubscribe button on each and every pull-request and issue where I'm mentioned by mistake.

**Reply by dgsus** [reply]
oops ;)
